src/data/uma-nickname-requirements.csv

```diff
@@ -77,6 +77,7 @@
 "メジロの冠名がつくウマ娘にGⅠで1回以上勝利し、\n菊花賞、宝塚記念、有馬記念で勝利して、\nファン数が240000人以上になる","<size=22>Win a GI race against a Mejiro at least once\nWin the Kikka Sho, Takarazuka Kinen, and Arima Kinen\nEarn a total of ≥240000 fans</size>"
 "ユニコーンS、安田記念を含むマイルの重賞で5勝以上し、\nうち1勝以上は天候（雨）で勝利かつ、\nマイルCSを5バ身差以上で勝利する","<size=22>Win a big race ≥5 times including the Unicorn Stakes, and Yasuda Kinen with one of them in rainy weather\nWin the Mile Championship with difference of ≥5 lengths</size>"
 "GⅠを1番人気で6回以上出走し、\nマイルCS、安田記念、有馬記念で勝利して\nファン数が240000人以上になる","<size=22>Participate a GI race ≥6 times being the 1st popular\nWin the Mile Championship, Yasuda Kinen, and Arima Kinen\nEarn a total of ≥240000 fans</size>"
+"作戦「追込」かつ1番人気で重賞6連勝以上、\nうち2戦以上はラストの直線で5人以上を抜いて勝利する","<size=22>As a chaser being the number one popular, win ≥6 consecutive big races,\nwin ≥2 of them by overtaking ≥5 other runners on the last straights</size>"
```

Comment:
I'd reword the last bit of this if possible space-wise to read:

"[...] win ≥6 consecutive big races, with overtaking ≥5 other runners on the last straight for ≥2 of them"